src/components/UTIL_PageMessageError.component-meta.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexComponent xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>43.0</apiVersion>
```

Comment:
This should be api v46. Also, the <packageVersions> section should be deleted for a new component, especially because this component doesn't actually reference any of the other packages directly.